homeassistant/components/overkiz/climate_entities/hitachi_air_to_air_heat_pump_ovp.py

```diff
@@ -0,0 +1,351 @@
+
+        return None
+
+    @property
+    def current_temperature(self) -> int | None:
+        """Return current temperature."""
+        if (state := self.device.states[ROOM_TEMPERATURE_STATE]) and state.value_as_int:
+            return state.value_as_int
+
+        return None
+
+    async def async_set_temperature(self, **kwargs: Any) -> None:
+        """Set new temperature."""
+        temperature = cast(float, kwargs.get(ATTR_TEMPERATURE))
+        await self._global_control(target_temperature=int(temperature))
+
+    @property
+    def preset_mode(self) -> str | None:
+        """Return the current preset mode, e.g., home, away, temp."""
+        if (
+            state := self.device.states[OverkizState.CORE_HOLIDAYS_MODE]
+        ) and state.value_as_str:
+            if state.value_as_str == OverkizCommandParam.ON:
+                return PRESET_HOLIDAY_MODE
+
+            if state.value_as_str == OverkizCommandParam.OFF:
+                return PRESET_NONE
+
+        return None
+
+    async def async_set_preset_mode(self, preset_mode: str) -> None:
+        """Set new preset mode."""
+        if preset_mode == PRESET_HOLIDAY_MODE:
+            await self.executor.async_execute_command(
+                OverkizCommand.SET_HOLIDAYS,
+                OverkizCommandParam.ON,
+            )
+        if preset_mode == PRESET_NONE:
+            await self.executor.async_execute_command(
+                OverkizCommand.SET_HOLIDAYS,
+                OverkizCommandParam.OFF,
+            )
+
+    # OVP has this property to control the unit's timer mode
+    @property
+    def auto_manu_mode(self) -> str | None:
+        """Return auto/manu mode."""
+        if (state := self.device.states[AUTO_MANU_MODE_STATE]) and state.value_as_str:
+            return state.value_as_str
+        return None
+
+    # OVP has this property to control the target temperature delta in auto mode
+    @property
+    def temperature_change(self) -> int | None:
+        """Return temperature change state."""
+        if (
+            state := self.device.states[TEMPERATURE_CHANGE_STATE]
+        ) and state.value_as_int:
+            return state.value_as_int
+
+        return None
+
+    @property
+    def min_temp(self) -> float:
+        """Return the minimum temperature."""
+        if self.hvac_mode == HVACMode.AUTO:
+            return TEMP_AUTO_MIN
+        return TEMP_MIN
+
+    @property
+    def max_temp(self) -> float:
+        """Return the maximum temperature."""
+        if self.hvac_mode == HVACMode.AUTO:
+            return TEMP_AUTO_MAX
+        return TEMP_MAX
+
+    def _control_backfill(
+        self, value: str | None, state_name: str, fallback_value: str
+    ) -> str:
+        """Overkiz doesn't accept commands with undefined parameters. This function is guaranteed to return a `str` which is the provided `value` if set, or the current device state if set, or the provided `fallback_value` otherwise."""
```

Comment:
Please reformat this to a short summary, followed by a blank line, then multiple lines with detailed explanation, maybe:
```suggestion
        """Return a command which will be accepted by Overkiz.

        Overkiz doesn't accept commands with undefined parameters. This function
        is guaranteed to return a `str` which is the provided `value` if set, or
        the current device state if set, or the provided `fallback_value` otherwise.
        """
```